### PR TITLE
[fix bug 1222181] Update PB CTA URL for Android/iOS.

### PR DESCRIPTION
**Pull request by jpetto**
- Fix bug on Windows where download button shows for
  up to date Fx users.

- Fix bug on Windows where two download buttons show
  for out of date Fx users.

### CODE REVIEW

File: media/js/base/global.js [hunk 1]
```diff
@@ -126,6 +126,13 @@ function isFirefoxMobile(userAgent) {
     return /Mobile|Tablet|Fennec/.test(ua);
 }
 
+// iOS does not follow same version numbers as desktop & Android, so may not be safe to
+// simply add this check to isFirefox. Will require further investigation/testing.
+function isFirefoxiOS(userAgent) {
```
Comment (alexgibson) [inline media/js/base/global.js]: Looks like this function is used in `/media/firefox/ios.js` too. Should we update that to use the global version here to?

**Comment by alexgibson** [comment]
Not introduced by this change, but I see a couple of linting errors when viewing the JS file locally. Worth tidying up here too?

`Line 5: site is defined but never used`
`Line 17: use ===`

**Comment by alexgibson** [comment]
New logic seems good, r+wc :wine_glass:

File: media/js/base/global.js (hunk 1 above)
Comment (jpetto) [inline media/js/base/global.js]: Yep - good call.

**Comment by jpetto** [comment]
Put the fixes in a separate commit for easy checking.

**Comment by alexgibson** [comment]
Changes look good, care to squash?

**Pull request merged**